chains/solana/contracts/tests/ccip/ccip_router_test.go

```diff
@@ -560,7 +566,7 @@ func TestCCIPRouter(t *testing.T) {
 			linkPool.PoolTokenAccount = addrLink
 			linkPool.User[linkPool.PoolSigner] = linkPool.PoolTokenAccount
 
-			ixAuth, err := tokens.SetTokenMintAuthority(token0.Program, token0.PoolSigner, token0.Mint, token0PoolAdmin.PublicKey())
+			ixAuth, err := tokens.SetTokenMintAuthority(token0.Program, token0Multisig.PublicKey(), token0.Mint, token0PoolAdmin.PublicKey())
```

Comment:
Please also add a multisig testcase to `tokenpool_test.go`, so we have some tests validating the behaviour in the actual burnmint pool and not just our test pool.

Reply:
Should we use the actual burnmint token pool program in the ccip_router_test.go ?

Reply:
Added a test in `pools_test.go`